common/resources.yaml

```diff
@@ -0,0 +1,51 @@
+groups:
+ - data_type_def: ExternalResources
+   data_type_inc: Container
+   doc: A pair of tables for tracking external resource references in a file
+   datasets:
+    - data_type_inc: Data
+      name: keys
+      doc: A table for identifying which objects in a file contain
+      dtype:
+       - name: key_name
+         dtype: text
+         doc: the UUID for the object that references the resourece
+
+    - data_type_inc: Data
+      name: resources
+      doc: A table for mapping user terms (i.e. keys) to resource entities
+      dtype:
+       - name: keytable_idx
+         dtype: uint
+         doc: foreign reference to KeyTable - the index for the key from KeyTable
+       - name: resource_name
+         dtype: text
+         doc: The name of the online resource (i.e. website, database) that has the entity
+       - name: resource_id
+         dtype: text
+         doc: The unique identifier for the resource entity at the resource
+       - name: uri
+         dtype: text
+         doc: the URI for the resource entity this reference applies to
+
+    - data_type_inc: Data
+      name: objects
+      doc: A table for mapping user terms (i.e. keys) to resource entities
+      dtype:
+       - name: object_id
+         dtype: text
+         doc: the UUID for the object
+       - name: field
+         dtype: text
+         doc: the field of the object
```

Comment:
```suggestion
         doc: The field of the object. This can be an empty string if the object is a dataset and the field is the dataset values.
```